### PR TITLE
Handling depth filters

### PR DESCRIPTION
**Pull request by pmbohm**
For https://github.com/aodn/backlog/issues/718
Reads the z-axis information from the getMetadata request and creates a depth/elevation filter if required. 
The selected z-axis is applied to the map by adding "ELEVATION=[a value]" to getMap requests.

### CODE REVIEW

**Comment by ggalibert** [comment]
To add to @anguss00 first point and for clarity, my concern is that the [getmetadata](http://geoserver-123.aodn.org.au/geoserver/ncwms?layerName=csiro_cars_monthly_url%2FTEMP&REQUEST=GetMetadata&item=layerDetails) returns ```"zaxis":{"values":[0,-50,-200,-1000,-5000],"positive":false,"units":"m"}``` while in the NetCDF file we have:
```
...
	short DEPTH(DEPTH) ;
		DEPTH:axis = "Z" ;
		DEPTH:long_name = "depth" ;
		DEPTH:positive = "down" ;
		DEPTH:reference_datum = "sea surface" ;
		DEPTH:standard_name = "depth" ;
		DEPTH:units = "m" ;
		DEPTH:valid_max = 12000s ;
		DEPTH:valid_min = 0s ;
...
 DEPTH = 0, 50, 200, 1000, 5000 ;
```
How did the getmetadata end up giving negative values? Is this because of the attribute ```"positive":false```? If so, then this attribute may need to be taken into account so that you can handle values that are of a consistent sign with what is stored in the dataset.

**Comment by pmbohm** [comment]
@anguss00 review changes pushed

File: web-app/js/portal/details/NcWmsPanel.js [hunk 1]
```diff
@@ -169,11 +168,17 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
...
-            return String.format("{0}-{1}",
-                this.zAxisFromPicker.getValue(),
-                this.zAxisToPicker.getValue()
-            );
+
+        if (this._isZAxisFilterAvailable()) {
+
+            if (this.layer.extraLayerInfo) {
+
+                var multiplier =  (this.layer.extraLayerInfo.zaxis.positive === false) ? -1 : 1;
```
Comment (anguss00) [inline web-app/js/portal/details/NcWmsPanel.js]: Sorry to be that guy, but there's an extra space here after the =

**Comment by anguss00** [comment]
One annoying comment, looks great aside from that

**Comment by anguss00** [comment]
Can you squash the commits?... lol jks

**Pull request merged**